src/main/java/com/kakao/golajuma/vote/web/controller/CloseVoteController.java

```diff
@@ -0,0 +1,27 @@
+package com.kakao.golajuma.vote.web.controller;
+
+import com.kakao.golajuma.auth.web.support.Login;
+import com.kakao.golajuma.common.support.respnose.ApiResponse;
+import com.kakao.golajuma.common.support.respnose.ApiResponseBody;
+import com.kakao.golajuma.common.support.respnose.ApiResponseGenerator;
+import com.kakao.golajuma.common.support.respnose.MessageCode;
+import com.kakao.golajuma.vote.domain.service.CloseVoteService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+public class CloseVoteController {
+	private final CloseVoteService closeVoteService;
+
+	// 투표 마감
+	@PostMapping("/vote/{voteId}/close")
+	public ApiResponse<ApiResponseBody.SuccessBody<Void>> closeVote(
+			@PathVariable int voteId, @Login Long userId) {
```

Comment:
여기선 voteId가 Long 타입이여야 할 것 같아요~

```suggestion
			@PathVariable Long voteId, @Login Long userId) {
```